run_e2e.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+source docker/env
+
+cd tests/e2e
+
+cargo run -- --base-path /home/filip/CloudStation/aleph/aleph-node/docker/data --account-ids $DAMIAN $TOMASZ $ZBYSZKO $HANSU --sudo-account-id $DAMIAN
```

Comment:
I'm not sure if I have such a base-path on my PC ;)

Reply:
:/